packages/ecc-client-ga4gh-trs/webpack.dev.cjs

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+const merge = require("webpack-merge");
+const baseConfig = require("./webpack.common.cjs");
+
+module.exports = merge(baseConfig, {
+    devServer: {
+        open: true,
+        port: 3000,
+    },
+    devtool: "inline-source-map",
```

Comment:
There are a few options you might want to explore here, depending on how your application is performing and a few other tradeoffs. Check [here](https://webpack.js.org/configuration/devtool/) to explore your options